source/includes/_fund_acceptance.md

```diff
@@ -1576,3 +1600,16 @@ function PaycheckoutSlide({
 ```
 
 <p class="lottie" id="payment-link-demo-right-stick"></p>
+
+## Fund Acceptance Response Codes
+
+Below is the list of HTTP Status Code for API Fund Acceptance:
+HTTP Status Code | State | Description
+---------- | ------- | -------
+200 | Final | Response success without error
+403 | Final | Forbidden (IP address is not whitelisted or request is deemed suspicious e.g SQL injection or XSS)
+404 | Final | Not Found (wrong URL)
+429 | Final | Request Rejected (Too Many Request to specific endpoint)
+500 | Non Final | Internal Server Error (OY! system encountered unknown error)
+503 | Final | Service Unavailable (OY! system is unable to process the request temporarily)
```

Comment:
Hmmm ini Final tapi deskripsinya "unable to process the request temporarily". Ini di docs yg lain juga kayak gini  Ga?

Reply:
eh tapi ini gua liat dari @nicosetyawanh juga pas bikin yang payment routing, yg RC 503 juga final tapi unable to process temporarily. ini dpt dari infra yaa?